src/System/Status/StatusChecker.php

```diff
@@ -560,7 +564,7 @@ private static function getPlaintextOutput(array $status): string
                 $output .= "GLPI_DBSLAVE_{$num}_{$slave_info['status']}\n";
             }
         } else {
-            $output .= "No slave DB\n";
+            $output .= "No slave DB\n"; // Leave as "slave" since plain text is already deprecated
```

Comment:
Then, maybe just:
```suggestion
            $output .= "No DB replica\n";
```

Reply:
Changing the text would be a BC break if people are still monitoring using the plain-text output.
Not sure how much people actually use it currently and would be affected.

Reply:
ok, yes